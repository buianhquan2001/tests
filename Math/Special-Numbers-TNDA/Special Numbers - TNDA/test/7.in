99
2732 2799
3719 2911
4473 2616
1428 4602
3724 4220
1186 4250
4997 1742
3667 4973
742 4279
4985 1927
1654 3645
1217 2518
3147 4251
3742 2007
4960 3807
2465 2584
4303 3077
3151 2398
3193 4744
2706 625
971 2323
1110 4962
2449 3841
1217 4546
881 3894
1172 1728
1558 1805
2840 3827
554 2707
548 2638
3234 992
1013 4956
731 1582
4625 3946
996 3690
3771 3654
2954 4109
2868 1480
1760 1620
1011 2520
737 3870
3900 3274
1062 2362
2823 2796
2957 2442
4210 1955
3312 4610
1322 4803
4439 2771
555 2276
1852 4232
776 1023
1923 3159
4498 2140
760 4288
2893 1268
4942 1318
4658 3760
4343 4687
2671 3738
3523 1384
1246 4113
845 4374
4046 1913
1654 1840
1264 1929
4333 3961
680 3413
1650 1540
535 3382
4777 3160
2360 2977
4138 2673
3137 4596
4662 2782
2417 4727
567 1674
3316 3443
4379 4540
977 968
2414 844
3605 2397
1793 4894
3211 4068
4423 2033
1726 2680
1747 880
1781 1568
2774 3405
2278 4692
1056 2023
3206 3430
2649 2861
4442 1290
2877 3588
3543 1240
2483 3246
4062 3943
985 2138
