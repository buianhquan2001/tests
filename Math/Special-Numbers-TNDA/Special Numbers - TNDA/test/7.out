900927679
113712570
913343286
818021791
600257537
324385989
579412005
342345961
948375659
183706156
390534462
353812980
692494118
309064310
927921470
286511433
336209697
345804939
939535970
17166892
573257686
162689534
413663705
544902789
554681009
163628504
246019542
420358549
587255569
919059194
582606428
552335664
686883644
581942948
480959207
219711383
484210261
525235554
867645716
164636257
923396875
713615483
187432179
653160347
921855394
764626771
225465526
23251668
257257154
414116223
925684297
70923691
453942188
982206057
558926294
458075784
867877764
769891675
85903901
140094040
941845894
863633015
524121332
389006503
853935361
310295983
328633629
751105688
44423862
870901967
200157356
220067075
639235654
829797591
901219688
587005493
538805829
224923985
797681021
524783039
270556730
393625492
955494360
559351463
532479598
62080882
694872473
384575956
988385752
486863840
842151102
775814949
197044343
504549911
447663318
936163774
581822984
716302284
674318421
